diff --git a/node_modules/@ckb-lumos/base/lib/utils.js b/node_modules/@ckb-lumos/base/lib/utils.js
index b0886e0..54b1c0f 100644
--- a/node_modules/@ckb-lumos/base/lib/utils.js
+++ b/node_modules/@ckb-lumos/base/lib/utils.js
@@ -48,7 +48,7 @@ function readBigUInt64LE(hex) {
 }
 
 const U128_MIN = BigInt(0);
-const U128_MAX = BigInt(2) ** BigInt(128) - BigInt(1);
+const U128_MAX = BigInt(340282366920938463463374607431768211455);
 function toBigUInt128LE(u128) {
   if (u128 < U128_MIN) {
     throw new Error(`u128 ${u128} too small`);
